### PR TITLE
Add Footer component

### PR DESCRIPTION
**Pull request by Eleein**
-Create Footer compoent
-Make Footer stick to bottom

Reported-by: Dania Duany

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,10 +2,12 @@ import React from "react";
 import { Route, Switch } from "react-router-dom";
 import { Home } from "./Home/Home";
 import { Header } from "./Header/Header";
+import {Footer} from "./zFooter/Footer";
```
Comment (joe-crick) [inline src/App.js:5]: This (`zFooter/Footer`) really does look bad, and people, if they review your GitHub, will wonder why you've done this. 😄

File: src/styles/global.scss [hunk 2]
```diff
@@ -138,6 +140,12 @@ body {
 .catch-phrase{
   font-family: Zilla Slab, san-serif, serif;
   font-style: italic;
-  color: white;
+  color: white
```
Comment (joe-crick) [inline src/styles/global.scss:143]: Why did you remove the semi-colon? While you can technically do this, it can lead to errors. I'd restore it, unless there's a good reason 😄

File: src/styles/global.scss [hunk 3]
```diff
@@ -138,6 +140,12 @@ body {
...
 
 }
+
+.App > div:first-child {
+  padding-bottom: 1.5rem;
```
Comment (joe-crick) [inline src/styles/global.scss:148]: Why are you using `rem`s here?

File: src/zFooter/Footer.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styles from "./Footer.module.scss";
+
+export function Footer() {
+  return (
+    <div className={styles.redRibbon}>
+      <span className={styles.footerText}>
+        <Link className={styles.link} to="/impressum">
```
Comment (joe-crick) [inline src/zFooter/Footer.js:9]: I'd advise against using magic strings. While it is common in the FE world to see this, it can create myriad problems. Instead, you can use an JS object (also called a `pojo`) that has your string constants in it, for example:

```javascript
export const appPaths = {
  impressum: "impressum",
  contact: "kontakt",
  // ...
}
```

For more information, see: https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings

File: src/zFooter/Footer.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+          Impressum
+        </Link>
+      </span>
+      <span className={styles.footerText}>
+        <Link className={styles.link} to="/kontakt">
```
Comment (joe-crick) [inline src/zFooter/Footer.js:14]: Magic strings, as above.

File: src/zFooter/Footer.module.scss [hunk 6]
```diff
@@ -0,0 +1,29 @@
+.redRibbon{
+
```
Comment (joe-crick) [inline src/zFooter/Footer.module.scss:2]: Extra space - consider removing it,

File: src/zFooter/Footer.module.scss [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  background-color: #cb463e;
```
Comment (joe-crick) [inline src/zFooter/Footer.module.scss:3]: Rather than inlining colors, borders, fonts, etc., put all of them in a `theme` file, and then reference your theme colors by name. This makes updating these values both easier, and more reliable.

File: src/zFooter/Footer.module.scss [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  color: white;
```
Comment (joe-crick) [inline src/zFooter/Footer.module.scss:4]: Avoid using named CSS color values. Instead, either use HEX, or rgba (rgba tends to be more flexible).

**Review by joe-crick — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (Eleein) [reply src/App.js:5]: Ok, I'll fix it. I wish there was a way to keep the folders organized in an intuitive way. Footer being at the top of my file directory doesn't make me very happy 😒

File: src/styles/global.scss (hunk 2 above)
Comment (Eleein) [reply src/styles/global.scss:143]: Done 😺

File: src/styles/global.scss (hunk 3 above)
Comment (Eleein) [reply src/styles/global.scss:148]: I deleted this...I am though using it in the App.module.scss file to make it more responsive

File: src/zFooter/Footer.js (hunk 4 above)
Comment (Eleein) [reply src/zFooter/Footer.js:9]: Thanks for that 👍

Comment (Eleein) [reply src/zFooter/Footer.js:9]: Should I create the object in a separate folder or in the same folder as the footer? For now I'll keep it in the same folder

File: src/zFooter/Footer.module.scss (hunk 6 above)
Comment (Eleein) [reply src/zFooter/Footer.module.scss:2]: Done 👍

File: src/zFooter/Footer.module.scss (hunk 7 above)
Comment (Eleein) [reply src/zFooter/Footer.module.scss:3]: Done 🐤 😄

File: src/zFooter/Footer.module.scss (hunk 8 above)
Comment (Eleein) [reply src/zFooter/Footer.module.scss:4]: Also done 😎

File: src/styles/global.scss (hunk 2 above)
Comment (joe-crick) [inline src/styles/global.scss:143]: Consider avoiding the use of named CSS colours. Instead, rely on the colours from your colour palette (i.e., theme).

File: src/zFooter/Footer.js (hunk 4 above)
Comment (joe-crick) [reply src/zFooter/Footer.js:9]: That works for now 😄

**Review by joe-crick — CHANGES_REQUESTED** [review]

File: src/zFooter/Footer.js (hunk 4 above)
Comment (Eleein) [reply src/zFooter/Footer.js:9]: 💯

**Review by joe-crick — APPROVED** [review]
